src/js/components/Search.jsx

```diff
@@ -24,11 +54,16 @@ export class Search extends PureComponent {
           onSearchChange={(data, searchTerm) => (
             this.handleSearchChange(data, searchTerm)
           )}
+          onSearchClear={() => this.handleSearchChange(null)}
         />
         {/* render results table */}
         {this.state.showResults && <ResultsTable
           {...props}
+          changeColumnWidths={(columns, table) => this.handleCalculateColumnWidths(columns, table)}
```

Comment:
`...SussolReactTable.propTypes` doesn't SussolReactTable ensure its own PropTypes? Anyway, didn't know you could spread syntax like that!